projects/plugins/jetpack/extensions/blocks/cookie-consent/attributes.js

```diff
@@ -0,0 +1,43 @@
+import { __ } from '@wordpress/i18n';
+
+export default {
+	text: {
+		type: 'string',
+		source: 'html',
+		selector: 'p',
+		default: __(
+			'Privacy & Cookies: This site uses cookies. By continuing to use this website, you agree to their use. To find out more, including how to control cookies, see here: <a href=https://automattic.com/cookies/>Cookie Policy</a>.',
```

Comment:
I suspect that self-hosted sites would need a different link here. Maybe Atomic too?

Reply:
Yes, for the widget we allow custom links in there, and also take into account the page you can set as your privacy policy page in WordPress:
https://github.com/Automattic/jetpack/blob/3104e377ae2d45ad2d4ffadfd839cee47aa871df/projects/plugins/jetpack/modules/widgets/class-jetpack-eu-cookie-law-widget.php#L138-L139

This is something we could improve on in a follow-up PR I think.